config.h.cmake

```diff
@@ -0,0 +1,58 @@
+/* config.h.cmake */

```

Comment:
Similarly, can we generate this from config.h.in?

Reply:
config.h.in is useless from the perspective of cmake, as all of the entries in it are of the form `#undef ....`.

A separate pre-processing step would be needed to transform the file into something cmake can use.

Reply:
Does this work?
```cmake
file(READ "config.h.in" original_config_h_in)
string(REGEX REPLACE "#undef" "#cmakedefine" new_config_h_cmake "${original_config_h_in}")
file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/config.h.cmake" "${new_config_h_cmake}")
configure_file("${CMAKE_CURRENT_BINARY_DIR}/config.h.cmake" "${CMAKE_CURRENT_BINARY_DIR}/config.h")
```